gpu_ocean/SWESimulators/Common.py

```diff
@@ -479,34 +479,39 @@ class SWEDataArakawaC:
     We use h as cell centers
     """
     def __init__(self, gpu_stream, nx, ny, halo_x, halo_y, h0, hu0, hv0, \
-                 asymHalo=None):
+                 fbl=False):
```

Comment:
I think we should refactor this in a subsequent PR so that SWEDataArakawaCFBL becomes a class to handle the special case. Right now it is OK, but it should be on our todo-list somewhere

Reply:
Agree. Made card in trello